docs/Information_nodes.rst

```diff
@@ -70,6 +70,60 @@ commands::
 
     --gres=gpu:a100:<number> --reservation=DGXA100
 
+MIG
+^^^
+
+.. _mig_nodes:
+
+MIG (`Multi-Instance GPU <https://www.nvidia.com/en-us/technologies/multi-instance-gpu/>`_)
+is an NVIDIA technology allowing certain GPUs to be
+partitioned into multiple *instances*, each of which has a roughly proportional
+amount of compute resources, device memory and bandwidth to that memory.
+
+NVIDIA supports MIG on its A100 GPUs and allows slicing the A100 into up to 7
+instances. Although this can theoretically be done dynamically, the SLURM job
+scheduler does not support doing so in practice as it does not model
+reconfigurable resources very well. Therefore, the A100s must currently be
+statically partitioned into the required number of instances of every size
+expected to be used.
+
+The new ``cn-g`` series of nodes include A100-80GB GPUs. One third have been
```

Comment:
```suggestion
The ``cn-g`` series of nodes include A100-80GB GPUs. One third have been
```

I would remove the temporality here. Just make the documentation accurate for the years to come. ;)